src/stylesheets/components/modal/_modal.scss

```diff
@@ -9,11 +9,19 @@
   visibility: hidden;
   opacity: 0;
   margin-top: 0rem;
+  z-index: 10000;
```

Comment:
I'm ok with this for now,  but I think hard coding the z-index is probably something we should avoid doing.  We can use Sass to manage this via calculation or use custom css classes with z-indexes specified.  Or something.